website/docs/r/org_user.html.markdown

```diff
@@ -0,0 +1,165 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org_user"
+sidebar_current: "docs-vcd-resource-org-user"
+description: |-
+  Provides a vCloud Director Organization user. This can be used to create, update, and delete organization users.
+---
+
+# vcd\_org\_user
+
+Provides a vCloud Director Org User. This can be used to create, update, and delete organization users, including org administrators.
+
+Supported in provider *v2.4+*
+
+~> **Note:** Only `System Administrator` or `Org Administrator` users can create users.
+
+## Example Usage
+
+```hcl
+resource "vcd_org_user" "my-org-admin" {
+  org = "my-org"
+
+  name          = "my-org-admin"
+  description   = "a new org admin"
+  role          = "Organization Administrator"
+  provider_type = "INTEGRATED"
+  password      = "change-me"
+  is_enabled    = true
+}
+
+resource "vcd_org_user" "test_user_vapp_author" {
+  org = "datacloud"
+  
+  name              = "test_user_vapp_author"
+  password_file     = "org_user_pwd.txt"
+  full_name         = "test user vapp author"
+  description       = "Org user test_user_vapp_author"
+  role              = "vApp Author"
+  is_enabled        = true
+  take_ownership    = true
+  provider_type     = "INTEGRATED"
+  stored_vm_quota   = 20
+  deployed_vm_quota = 20
+  instant_messaging = "@test_user_vapp_author"
+  email_address     = "test_user_vapp_author@test.company.org"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to which the VDC belongs. Optional if defined at provider level.
+* `name` - (Required) A unique name for the user.
+* `password` - (Optional, but required if `password_file` was not given) The user password. This value is never returned 
+  on read. It is inspected on create and modify. On modify, the absence of this element indicates that the password 
+  should not be changed.
+* `password_file` (Optional, but required if `password` was not given). A text file containing the password. 
+  Using this property instead of `password` has the advantage that the sensitive data is not saved into Terraform state 
```

Comment:
Need a comment on how to update after changing the password in file.

Reply:
done